embulk-input-oracle/src/main/java/org/embulk/input/OracleInputPlugin.java

```diff
@@ -49,6 +49,14 @@
         @Config("password")
         @ConfigDefault("\"\"")
         public String getPassword();
+
+        @Config("tnsname_path")
```

Comment:
`tnsname_path` should be `tns_admin_path` because it isn't the path of the `tnsnames.ora` file but the path of the directory that contains the `tnsnames.ora` file and other files.